src/frontend/komponenter/Fagsak/Vedtak/VedtakContainer.tsx

```diff
@@ -72,27 +70,49 @@ const VedtakContainer: React.FC<IProps> = ({ behandling, fagsak }) => {
         behandling.behandlingsårsakstype ===
             Behandlingårsak.REVURDERING_FEILUTBETALT_BELØP_HELT_ELLER_DELVIS_BORTFALT;
 
-    const { hentSjekkLikhetPerioder } = useSjekkLikhetPerioder(behandling.behandlingId);
+    const { erPerioderLike, hentSjekkLikhetPerioder } = useSjekkLikhetPerioder(
+        behandling.behandlingId
+    );
+    const { erPerioderSlåttSammen, hentErPerioderSlåttSammen } = useErPerioderSlåttSammen(
+        behandling.behandlingId
+    );
+    const { slåSammenPerioder, feilmelding } = useSlåSammenPerioder(
+        behandling.behandlingId,
+        !erPerioderSlåttSammen
+    );
+
+    const handleSlåSammenPerioder = async () => {
+        await slåSammenPerioder();
+        await hentSjekkLikhetPerioder();
+        await hentErPerioderSlåttSammen();
```

Comment:
Trengs disse to, eller kunne vi kjørt en `settErPerioderSlåttSammen(true/false)` direkte?

Reply:
Vi har nå en state i frontend for å slippe `hentErPerioderSlåttSammen()`